drivers/clock_control_nrfxlib/Kconfig

```diff
@@ -1,14 +1,24 @@
 #
-# Copyright (c) 2018 Nordic Semiconductor ASA
+# Copyright (c) 2019 Nordic Semiconductor ASA
 #
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
-config CLOCK_CONTROL_NRF_LL_NRFXLIB
+#use depends on instead of if else and default y
+config CLOCK_CONTROL_BT_LL_NRFXLIB
 	bool "nRF BLE Controller clock control driver"
-	default y
+	depends on BT_LL_NRFXLIB
 	select CLOCK_CONTROL_NRF_FORCE_ALT
 
+config CLOCK_CONTROL_MPSL_NRFXLIB
+	bool "MPSL nRF clock control driver"
+	depends on MPSL_NRFXLIB
+	select CLOCK_CONTROL_NRF_FORCE_ALT
+
+config CLOCK_CONTROL_MPSL
```

Comment:
Should CLOCK_CONTROL_MPSL or CLOCK_CONTROL_MPSL_NRFXLIB be removed?

Reply:
Yes maybe. I don't think we need both. The reason why I added it was to use it here: https://github.com/sofu1/fw-nrfconnect-zephyr-1/pull/1

Reply:
Only using CLOCK_CONTROL_NRFXLIB now